examples/v2/chat2.nim

```diff
@@ -216,7 +216,7 @@ proc publish(c: Chat, line: string) =
       encodedPayload = payload.encode(version, c.node.rng[])
     if encodedPayload.isOk():
       var message = WakuMessage(payload: encodedPayload.get(),
-        contentTopic: c.contentTopic, version: version)
+        contentTopic: c.contentTopic, version: version, timestamp: Timestamp(time*1000*1000*1000))
```

Comment:
```suggestion
        contentTopic: c.contentTopic, version: version, timestamp: Timestamp(time*1000_000_000))
```

Reply:
You can use getNanosecondTime() for the conversion (and omit Timestamp type specification):

```suggestion
        contentTopic: c.contentTopic, version: version, timestamp: getNanosecondTime(time))
```